tests/foreman/ui/test_activationkey.py

```diff
@@ -379,15 +371,17 @@ def test_negative_create_activation_key_2(self):
         @BZ: 1177158
 
         """
-
-        name = gen_string("alpha", 10)
-        description = gen_string("alpha", 1001)
+        name = gen_string('alpha', 10)
+        description = gen_string('alpha', 1001)
         with Session(self.browser) as session:
-            make_activationkey(session, org=self.org_name,
-                               name=name, env=ENVIRONMENT,
-                               description=description)
-            self.assertIsNotNone(self.activationkey.wait_until_element
-                                 (common_locators["common_haserror"]))
+            make_activationkey(
+                session, org=self.org_name, name=name, env=ENVIRONMENT,
+                description=description)
+            self.assertIsNotNone(
+                self.activationkey.wait_until_element(
+                    common_locators['common_haserror']
+                )
```

Comment:
I have to update that make_activation key and place one argument on its own line.

Good catch.